bucket/snaptimer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "0.1",
+    "description": "Windows Countdown Timer",
+    "homepage": "https://dan.hersam.com/software/snaptimer/index.php",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://dan.hersam.com/software/license.php"
+    },
+    "url": "https://cdn.hersam.com/software/snaptimer/SnapTimer.zip",
+    "hash": "sha1:7131F4C79038B8059F8EF16601077F075CFAD2DC",
+    "extract_dir": "SnapTimer",
+    "pre_install": "if (!(Test-Path \"$persist_dir\\SnapTimer.ini\")) { New-Item \"$dir\\SnapTimer.ini\" | Out-Null }",
+    "shortcuts": [
+        [
+            "SnapTimer.exe",
+            "SnapTimer"
+        ]
+    ],
+    "persist": "SnapTimer.ini",
+    "checkver": "<b>Version:</b>\\s+([\\d.]+)",
+    "autoupdate": {
+        "url": "https://cdn.hersam.com/software/snaptimer/SnapTimer.zip",
+        "hash": {
+            "url": "https://dan.hersam.com/software/snaptimer/index.php",
+            "regex": "<b>SHA-1:</b>\\s+$sha1<br>"
+        }
+    }
```

Comment:
Latest release 2010

Where is version 0.2 mentioned in readme?

![image](https://user-images.githubusercontent.com/13260377/97678248-cb676e00-1a93-11eb-8148-96a0abff665a.png)


```suggestion
    "persist": "SnapTimer.ini"
```

Reply:
That's interesting. :<